### PR TITLE
Add Placeholder View Mode Without any data

### PR DESCRIPTION
**Pull request by nataliyast**
https://trello.com/c/eo6vStbm/808-entity-placeholder-view-form-with-empty-values-pom

### CODE REVIEW

**Review by guram21 — APPROVED** [review]

**Review by kgoldpasv — APPROVED** [review]

File: src/test/java/test/entity/EntityPlaceholderTest.java [hunk 1]
```diff
@@ -215,7 +215,7 @@ public void viewRecordTest() {
     public void createRecordWithoutAnyDataTest() {
 
         MainPage mainPage = new MainPage(getDriver());
-        currentUser = mainPage.getCurrentUser();
+       // currentUser = mainPage.getCurrentUser();
```
Comment (oooleks) [inline src/test/java/test/entity/EntityPlaceholderTest.java:218]: Please explain the purpose of committed commented code. If not needed just delete it (#218).

Comment (nataliyast) [reply src/test/java/test/entity/EntityPlaceholderTest.java]: Thank you. I changed it

**Review by guram21 — APPROVED** [review]

**Review by kgold2018 — APPROVED** [review]
OK

File: src/test/java/test/entity/EntityPlaceholderTest.java [hunk 2]
```diff
@@ -226,4 +225,22 @@ public void createRecordWithoutAnyDataTest() {
         Assert.assertEquals(placeholderPage.getRowCount(), 1);
         Assert.assertEquals(placeholderPage.getRowIconClass(0), AppConstant.RECORD_ICON_CLASS);
     }
+
+    @Test(dependsOnMethods = "createRecordWithoutAnyDataTest")
+    public void viewRecordWithoutAnyDataTest() {
```
Comment (oooleks) [inline src/test/java/test/entity/EntityPlaceholderTest.java:230]: Just a comment: this test is exact copy of viewRecordTest(). So you would better to include these asserts for Table page into createRecordWithoutAnyDataTest() which currently does not have any data validations but row count only, hence less code, faster execution, same validation.

File: src/test/java/test/entity/EntityPlaceholderTest.java [hunk 3]
```diff
@@ -226,4 +225,22 @@ public void createRecordWithoutAnyDataTest() {
...
+
+        MainPage mainPage = new MainPage(getDriver());
+        currentUser = mainPage.getCurrentUser();
```
Comment (oooleks) [inline src/test/java/test/entity/EntityPlaceholderTest.java:233]: No need to assign the same value which has been already assigned to this variable `currentUser` earlier in createRecordWithoutAnyDataTest()

**Pull request merged**